src/assets/i18n/de.json

```diff
@@ -120,22 +120,29 @@
   },
   "tracking": {
     "title": "Standortverlauf",
-    "toggleTitle": "Aufzeichnung starten / stoppen",
-    "stateRunning": "Läuft",
-    "stateStopped": "Gestoppt",
+    "toggleOn": "Aufzeichnung starten",
+    "toggleOff": "Aufzeichnung stoppen",
+    "togglePauseSubtitle": "Wähle hier die Länge der Pause",
+    "stateRunning": "Standortaufzeichnung ist aktuell aktiv",
+    "stateStopped": "Standortaufzeichnung ist aktuell gestoppt",
```

Comment:
I would ditch the "aktuell" in those strings, because it seems kind of redundant in this sentence. This is personal preference though, please skip if you like ;)